[2024-09-10T08:23:14.822+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T08:23:14.890+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V5.second_task scheduled__2024-04-03T00:00:00+00:00 [queued]>
[2024-09-10T08:23:14.916+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V5.second_task scheduled__2024-04-03T00:00:00+00:00 [queued]>
[2024-09-10T08:23:14.920+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T08:23:14.964+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2024-04-03 00:00:00+00:00
[2024-09-10T08:23:14.988+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=40418) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T08:23:15.005+0000] {standard_task_runner.py:72} INFO - Started process 40441 to run task
[2024-09-10T08:23:14.996+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V5', 'second_task', 'scheduled__2024-04-03T00:00:00+00:00', '--job-id', '9338', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpcsreb8lg']
[2024-09-10T08:23:15.010+0000] {standard_task_runner.py:105} INFO - Job 9338: Subtask second_task
[2024-09-10T08:23:15.243+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V5.second_task scheduled__2024-04-03T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T08:23:15.475+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_V5' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-04-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-03T00:00:00+00:00'
[2024-09-10T08:23:15.477+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T08:23:15.514+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T08:23:15.519+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the second task!']
[2024-09-10T08:23:15.535+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T08:23:15.538+0000] {subprocess.py:93} INFO - hey, this is the second task!
[2024-09-10T08:23:15.539+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T08:23:15.604+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T08:23:15.608+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V5, task_id=second_task, run_id=scheduled__2024-04-03T00:00:00+00:00, execution_date=20240403T000000, start_date=20240910T082314, end_date=20240910T082315
[2024-09-10T08:23:15.698+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T08:23:15.712+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
